delft3dworker/serializers.py

```diff
@@ -59,23 +59,43 @@ class SceneSerializer(serializers.ModelSerializer):
     class Meta:
         model = Scene
         fields = (
```

Comment:
Well, the whole point of this fix was to minimise data exchanged between BE and FE: I'd like the adding of data sent to the FE to be explicit and thus, a deliberate choice by the dev (instead of a bug because the dev forgot to filter a new data-heavy model property).